k8s/server_deployment.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1beta1
```

Comment:
How can we reach this server that is deployed inside of Kubernetes cluster? Please look into https://kubernetes.io/docs/tutorials/stateless-application/expose-external-ip-address/ and https://kubernetes.io/docs/concepts/services-networking/service/